### PR TITLE
correct invalid urls and close issue

### LINKED ISSUE #85495: 404 urls
**What happened**:
404 urls
**What you expected to happen**:
fix them

1. staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md,https://github.com/kubernetes-sigs/kustomize/tree/master/pkg/transformers/config/defaultconfig
1. staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md,https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/transformers/config/defaultconfig/commonannotations.go
1. staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md,https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/transformers/config/defaultconfig/commonlabels.go
1. staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md,https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/transformers/config/defaultconfig/namereference.go

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

correct invalid urls and close issue

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/kubernetes/issues/85495

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Comment by tanjunchen** [comment]
/cc @neolit123  
update the urls and  close issue. Thanks

**Review by neolit123 — COMMENTED** [review]
/sig cli

**Comment by tanjunchen** [comment]
/cc @neolit123  
for long time , please help review it , Thanks.  :))

**Comment by neolit123** [comment]
/uncc
@tanjunchen 
try asking in the #sig-cli channel on slack.

**Comment by tanjunchen** [comment]
> /uncc
> @tanjunchen
> try asking in the #sig-cli channel on slack.

Sure

File: staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md [hunk 1]
```diff
@@ -565,6 +565,7 @@ Meta Options control how Kustomize generates and transforms Resource Config.
 
 `configurations` is used to configure the built-in Kustomize Transformers to work with CRDs.  The built-in
 Kustomize configurations can be found [here](https://github.com/kubernetes-sigs/kustomize/tree/master/pkg/transformers/config/defaultconfig)
+https://github.com/kubernetes-sigs/kustomize/tree/v3.3.1/pkg/transformers/config/defaultconfig
```
Comment (surajnarwade) [inline staging/src/k8s.io/kubectl/docs/book/pages/reference/kustomize.md]: I think, this url should go inside bracket  after `here`

**Comment by tanjunchen** [comment]
@surajnarwade done.

**Review by soltysh — APPROVED** [review]
/approve
Leaving final voice for @monopole or @Liujingfang1

**Comment by tanjunchen** [comment]
kindly ping 
/cc @monopole

**Pull request closed without merging**